services/src/main/java/org/keycloak/services/clientpolicy/executor/SecureClientAuthEnforceExecutorFactory.java

```diff
@@ -56,6 +54,25 @@ public void init(Scope config) {
 
     @Override
     public void postInit(KeycloakSessionFactory factory) {
+        ProviderConfigProperty isAugmentProperty = new ProviderConfigProperty(
+                IS_AUGMENT, "Augment Configuration", "If On, then the during client creation or update, the configuration of the client will be augmented to enforce usage of PKCE",
```

Comment:
"enforce usage of PKCE" might be "enforce the authentication method to new clients" ?